challenges/open_research_challenge.tex

```diff
@@ -0,0 +1,62 @@
+% !TeX root = ../SPL-Challenges.tex
+% !TeX spellcheck = en_US
+\section{Open Research Challenge}
+The idea of this challenge is to create a platform within the Standard Platform League for teams to showcase innovations that contribute to the growth, development, and improvement of the league, beyond the core competition objectives. These innovations aim to enrich the league's ecosystem by fostering collaboration, technological advancement, and accessibility.
+
+This challenge incentivizes teams to focus on league-wide growth areas such as software, hardware, and community alignment. It provides an opportunity for teams to make impactful contributions in areas like fostering innovation through creative solutions, strengthening collaboration within the SPL community, enhancing accessibility for new and existing teams, boosting engagement with spectators and sponsors, and supporting the long-term growth and sustainability of the league.
+
+\subsection{Submission Requirements}
+Teams must submit a brief description of their planned contribution by \textbf{July 1, 2025} to the TC (\url{rc-spl-tc@lists.robocup.org}).
+The description should include:
+    \begin{itemize}
+        \item \textbf{Objective}: The purpose and scope of the innovation.
+        \item \textbf{Expected Impact}: How it benefits SPL or improves existing systems.
+        \item \textbf{Implementation Feasibility}: Resources and time needed to accomplish/execute the contribution.
+    \end{itemize}
+
+\subsection{Presentation Format}
+Contributions will be presented through a \textbf{10-minute presentation} during the SPL competition.
+Presentations must include:
+\begin{itemize}
+    \item Clear explanation of the contribution and its impact.
+    \item A \textbf{live demonstration}, if feasible, or a \textbf{video demonstration} otherwise.
+\end{itemize}
+The presentations will also be uploaded to the \textbf{SPL website} for public reviewing and reference.
```

Comment:
Maybe add to share to TC the day of presentation to get in the schedule ? Otherwise I feel we will need to track them.